src/tests/Microsoft.PowerFx.Core.Tests.Shared/ExpressionTestCases/AsType_UO.txt

```diff
@@ -0,0 +1,151 @@
+#SETUP: AllowTypeLiteral
+
+// Primitives
+>> AsType(ParseJSON("987654321"), Number)
+987654321
+
+>> AsType(ParseJSON("98765.4321"), Decimal)
```

Comment:
>98765.4321 [](http://example.com/codeflow?start=21&length=10)

For decimal, can you use a value that cannot be represented as a float? For example, `12345678901234.567890123456`